internal/mempool/v1/mempool.go

```diff
@@ -101,6 +109,12 @@ func NewTxMempool(
 		txStore:       NewTxStore(),
 		gossipIndex:   clist.New(),
 		priorityIndex: NewTxPriorityQueue(),
+		heightIndex: NewWrappedTxList(func(wtx1, wtx2 *WrappedTx) bool {
+			return wtx1.height >= wtx2.height
+		}),
+		timestampIndex: NewWrappedTxList(func(wtx1, wtx2 *WrappedTx) bool {
+			return wtx1.timestamp.After(wtx2.timestamp) || wtx1.timestamp.Equal(wtx2.timestamp)
```

Comment:
Is the second part, i.e `|| wtx1.timestamp.Equal(wtx2.timestamp)` actually needed. If they are the same then it doesn't matter which order they end up in

Reply:
Yeah, this is just to mimic `>=` but for `time.Time` types.